### ISSUE TITLE
"labels" (i.e. additional information) in the training step, that are NOT intrinsic star properties

### ISSUE THREAD

**Issue by HWRix** [mkness/TCS#4]
This is just to commit to memory a thought on a possible slight conceptual extension of the Cannon.

**Background/motivation**: Galah, serving as an example here, is a survey, where one of our basic
assumptions is known to fail: the LSF is the same for all spectra. The LSF, as I qualitatively understand it, varies systematically & predictably with distance R of the fiber from the plate center. 
So, let's presume we have a good set of reference objects that cover "label space" (logg, Teff, [X/H]) reasonably well, and also have an "experimental set-up label" R (with some nice coverage).
Let's also presume that for all survey objects in the test step, this distance R is also known.

**Implication for the training step**:
I would suspect, in the training step, "R" simply is treated as a label.

**Implications for the test step**:
Here, "R" becomes an known, fixed input (for each survey object), and differs from the bona-fide
labels, as it is simply not solved/optimized for. 

In a generalized, Bayesian Cannon, R is simply a label which has a delta-function prior for each object;
so could be treated more analogous..

For down the road .. this is presumably is just one example of the broader idea that there are "pieces of information" that can be treated like labels in the training step, but aren's actually labels for the purposes of the test step.

**Comment by davidwhogg** [comment]
This all sounds correct to me -- and necessary, unless we smooth to common resolution (LSF).

**Comment by davidwhogg** [comment]
A few realizations:
- Isn't this an issue for TheCannon or AnnaCannon? not TCS?
- I now (contra 3 days ago) don't think this is the right approach -- really we want training data _at_ every LSF value.
- This means either re-take training data at every LSF value, or else take it at good LSF values and then degrade the LSF artificially.
